src/rpc/net.cpp

```diff
@@ -81,37 +87,40 @@ UniValue getpeerinfo(const UniValue& params, bool fHelp)
-            "    \"bytessent\": n,            (numeric) The total bytes sent\n"
-            "    \"bytesrecv\": n,            (numeric) The total bytes received\n"
-            "    \"conntime\": ttt,           (numeric) The connection time in seconds since epoch (Jan 1 1970 GMT)\n"
-            "    \"timeoffset\": ttt,         (numeric) The time offset in seconds\n"
-            "    \"pingtime\": n,             (numeric) ping time\n"
-            "    \"pingwait\": n,             (numeric) ping wait\n"
-            "    \"version\": v,              (numeric) The peer version, such as 170002\n"
-            "    \"subver\": \"/MagicBean:x.y.z[-v]/\",  (string) The string version\n"
-            "    \"inbound\": true|false,     (boolean) Inbound (true) or Outbound (false)\n"
-            "    \"startingheight\": n,       (numeric) The starting height (block) of the peer\n"
-            "    \"banscore\": n,             (numeric) The ban score\n"
-            "    \"synced_headers\": n,       (numeric) The last header we have in common with this peer\n"
-            "    \"synced_blocks\": n,        (numeric) The last block we have in common with this peer\n"
+            "    \"id\": n,                              (numeric) peer index\n"
+            "    \"addr\": \"host:port\",                (string) the ip address and port of the peer\n"
+            "    \"addrlocal\": \"ip:port\",             (string) local address\n"
+            "    \"services\":\"xxxxxxxxxxxxxxxx\",      (string) the services offered\n"
+            "    \"tls_established\": true|false,        (boolean) status of TLS connection\n"
+            "    \"tls_verified\": true|false,           (boolean) status of peer certificate. True if the chain of trust of a peer certificate can be verified using the OS certificate store\n"
+            "    \"lastsend\": ttt,                      (numeric) the time in seconds since epoch (Jan 1 1970 GMT) of the last send\n"
+            "    \"lastrecv\": ttt,                      (numeric) the time in seconds since epoch (Jan 1 1970 GMT) of the last receive\n"
+            "    \"bytessent\": n,                       (numeric) the total bytes sent\n"
+            "    \"bytesrecv\": n,                       (numeric) the total bytes received\n"
+            "    \"conntime\": ttt,                      (numeric) the connection time in seconds since 1 Jan 1970 GMT\n"
+            "    \"timeoffset\": ttt,                    (numeric) the time offset in seconds\n"
+            "    \"pingtime\": n,                        (numeric) ping time\n"
+            "    \"pingwait\": n,                        (numeric) ping wait\n"
+            "    \"version\": v,                         (numeric) the protocol version of the peer\n"
+            "    \"subver\": \"/MagicBean:x.y.z[-v]/\",  (string) the user agent of the peer\n"
```

Comment:
I am not sure of this. If we execute getpeerinfo via cli we have:
[...]
   "version": 170002,
   "subver": "/zen:2.0.23/",
[...]

Reply:
Addressed in https://github.com/HorizenOfficial/zen/pull/379/commits/728008a9b05269c2e338dfb33c11831aa24d32f6